ap.md

```diff
@@ -0,0 +1,236 @@
+{::options parse_block_html="true" /}
+
+<div class="row">
+# Build Your Own Internet
+<div class="large-8 columns">
+
+Along with connecting to an existing Wifi network, Tessel 2 can also act as a portable router and emit a custom Wifi network called an access point. We can use this functionality to connect directly with our Tessel through a web browser. The advantage of doing this through Tessel's access point instead of through a local Wifi network is reduced latency of requests to the Tessel since they won't be making a detour through a different router. This also means there is no need for a local network for connecting to Tessel through the web. By the end of this article, we'll be able to control the Tessel LEDs through a web app served by the Tessel.  
```

Comment:
Hm, it's a good try but at the end I still don't really understand what this is or why I would want it. Can you give an example? Other questions: do you need Tessel to be connected to the internet in order to do this? Could it replace your home router? WAPs and routers are somewhat mysterious and confusing black boxes, might be helpful to link out to further reading somewhere (wikipedia?) that explains what the different parts are.

Or better yet, replace the image on the right with a diagram: what's the process we're currently using, and how can we cut out the middleman by making Tessel an AP?

With the added challenge, of course, that this section needs to be as short as possible/broken up into short paragraphs in order to get people to actually read it.

I'm happy to work with you back & forth on this one - it's probably the toughest introduction we have to do in the whole tutorial.